examples/prompting/Basic_Reasoning.ipynb

```diff
@@ -0,0 +1,239 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    Problem: """
</pre><p> </p><p>Putting a trailing clause at the end of the system instruction assumes that the system instruction is concatenated with the input contents. I don't know that we can or should assume that.</p><p> </p><p>If this works fine without this line, then remove it. If it's needed, move it to the input text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Just removed and it seems to be working fine. Thanks Mark!